lib/unparser/emitter/send.rb

```diff
@@ -199,6 +209,12 @@ def local_variable_clash?
         local_variable_scope.local_variable_defined_for_node?(node, selector)
       end
 
+      # Test if selector parses as constant
+      #
+      # @return [Boolean]
```

Comment:
To be consistent with all the other private methods in this class, I think this should specify `@api private`.